modules/gapi/test/cpu/gapi_imgproc_tests_fluid.cpp

```diff
@@ -16,6 +16,37 @@ namespace
 namespace opencv_test
 {
 
+INSTANTIATE_TEST_CASE_P(ResizeTestFluid, ResizeTest,
+                        Combine(Values(CV_8UC3/*CV_8UC1, CV_16UC1, CV_16SC1*/),
```

Comment:
Do extra test cases make sense?

Reply:
They do. I think they're commented because of limited Fluid functionality. They are implemented in OCV but are not currently working on Fluid. Whether they can actually be implemented - @anna-khakimova can help with advise I believe.  
Maybe they should be turned into `TODO` comment, placed not here but in `gfluidimgproc` next to Fluid-resize implementation.

Reply:
Need to add comment about it, from my point of view.

Reply:
Or a task can be created to extend test cases.